api/v1alpha1/extension_types.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Although this looks like a bool, there might
+type ExtensionManaged string
+
+const (
+	// Pause resolution of this Extension
+	ExtensionPausedTrue ExtensionManaged = "Active"
+	// Peform resolution of this Extension
+	ExtensionPausedFalse ExtensionManaged = "Paused"
```

Comment:
The names of these don't align with what I think our convention has been in the past regarding Condition naming and I feel this falls into a similar boat. I think something like:
```suggestion
	ExtensionManagedActive ExtensionManaged = "Active"
	// Peform resolution of this Extension
	ExtensionManagedPaused ExtensionManaged = "Paused"
```

is more indicative of these constants values and will, IMO, ultimately result in a better experience when reading and writing code that uses them.

Reply:
Yeah, I missed some renames :(